products/BellHybrid/apps/application-bell-onboarding/layouts/OnBoardingShortcutsLayoutClassic.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "OnBoardingShortcutsLayoutClassic.hpp"
+#include <apps-common/widgets/BellBaseLayout.hpp>
+#include <i18n/i18n.hpp>
+#include <Style.hpp>
+#include <ImageBox.hpp>
+#include <TextFixedSize.hpp>
+
+namespace
+{
+    namespace container
+    {
+        constexpr inline auto width      = 544U;
+        constexpr inline auto height     = 220U;
+        constexpr inline auto top_margin = 74U;
+
+        namespace separator
+        {
+            constexpr inline auto height = 22U;
+        } // namespace separator
+
+        namespace image
+        {
+            constexpr inline auto height = container::height - separator::height;
+            constexpr inline auto width  = 448U;
+        } // namespace image
+
+        namespace text
+        {
+            constexpr inline auto height = 146U;
+            constexpr inline auto width  = 448U;
+        } // namespace text
+
+    } // namespace container
+} // namespace
+
+namespace gui
+{
+    OnBoardingShortcutsLayoutClassic::OnBoardingShortcutsLayoutClassic(std::string image,
+                                                                       std::string description,
```

Comment:
Shouldn't strings be `const&`?

`Image::Image` and `Text::setRichText` take `UTF8` so these strings are going to be converted. Would it make sense to take `const UTF8&` here?

Reply:
Yes, it definitely should be `const&`, and also it'd be better to use `UTF8` instead of `string`. Thanks :)

Reply:
Done